src/m365/spo/commands/site/site-commsite-enable.ts

```diff
@@ -54,20 +64,36 @@ class SpoSiteCommSiteEnableCommand extends SpoCommand {
     this.validators.push(
       async (args: CommandArgs) => {
         if (args.options.designPackageId &&
-          !validation.isValidGuid(args.options.designPackageId)) {
-          return `${args.options.designPackageId} is not a valid GUID`;
+          !this.isValidDesignPackageId(args.options.designPackageId)) {
+          return `${args.options.designPackageId} is not a valid designPackageId. Allowed values are: ${Object.values(DesignPackage).join(', ')}.`;
+        }
+
+        if (args.options.designPackage) {
+          const designPackage = args.options.designPackage as keyof typeof DesignPackage;
+          if (!(designPackage in DesignPackage)) {
```

Comment:
Can't we use an undefined check here? It seems odd to check this after casting it in the line above.